src/main/java/frc/robot/OI.java

```diff
@@ -1,11 +1,51 @@
 package frc.robot;
 
+import edu.wpi.first.networktables.EntryListenerFlags;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+
+
 /**
  * This class is the glue that binds the controls on the physical operator
  * interface to the commands and command groups that allow control of the robot.
  */
 public class OI {
     public OI() {
+        createOperatorCommands();
+        NetworkTableEntry changeSettingsEntry = SmartDashboard.getEntry("isHillelSettings");
+        changeSettingsEntry.setBoolean(true);
+        // Listens for operator settings change. If it changes while the robot is disabled, the OI updates the button bindings.
+        changeSettingsEntry.addListener((entryNotification) -> {
```

Comment:
changeSettingsEntry.addListener((entryNotification) -> {
you dont have to (entryNotification):
        changeSettingsEntry.addListener(entryNotification -> {